genvm/vm_test.go

```diff
@@ -1441,9 +1442,13 @@ func runTestCases(t *testing.T, tcs []templateTestCase, genTester func(t *testin
 				}
 				for account, changes := range layer.expected {
 					prev, err := accounts.Get(tt.db, tt.accounts[account].getAddress(), lid.Sub(1))
-					require.NoError(tt, err)
+					if err != nil {
+						require.ErrorIs(t, err, sql.ErrNotFound)
+					}
```

Comment:
This assertion and the one below seems wrong. It looks like for every test it doesn't matter if the account exists or not?

Reply:
The problem is that previously the `Get()` would return a stub account if the account for given address doesn't exist and now it returns `ErrNotFound` instead. The test doesn't care if the account exists in the `DB`. Theoretically (logically?) all addresses have a "virtual" stub account with 0 nonce and balance.